source/contracts/Augur.sol

```diff
@@ -132,4 +133,9 @@ contract Augur is Controlled {
         TokensTransferred(_universe, msg.sender, _from, _to, _value);
         return true;
     }
+
+    function getProtectedTokens() internal returns (address[]) {
+        address[] memory _protectedTokens = new address[](0);
+        return _protectedTokens;
```

Comment:
Why two lines instead of one?  `return new address[](0);`?

Reply:
Will change. This just worked better as a copy-paste-review format when doing all the boilerplate.